src/itemPublished/itemPublished.ts

```diff
@@ -16,3 +22,17 @@ export interface PackedItemPublished {
   item: PackedItem;
   totalViews: number;
 }
+
+export class PublishableItemTypeChecker {
+  private static readonly ALLOWED_PUBLISHED_TYPES: ItemTypeUnion[] = [
+    ItemType.FOLDER,
+  ];
+
+  public static isItemTypeAllowedToBePublished(itemType: ItemTypeUnion) {
+    this.ALLOWED_PUBLISHED_TYPES.includes(itemType);
```

Comment:
Are you missing the return here ? I don't think JS has implicit returns (yet).

Reply:
Yes I saw a little bit too late ^^